test/autoschedulers/li2018/CMakeLists.txt

```
@@ -39,6 +40,7 @@ if (WITH_PYTHON_BINDINGS)
         list(TRANSFORM PYTHONPATH PREPEND "PYTHONPATH=path_list_prepend:")
 
         set_tests_properties(li2018_gradient_autoscheduler_test_py PROPERTIES
+                             DEPENDS Li2018
```

Is there a test named `Li2018`? Should this be the following?
```suggestion
                             DEPENDS li2018_test
```